bidding-impl/src/main/resources/application.conf

```diff
@@ -21,3 +21,6 @@ akka.actor.serialization-bindings {
 }
 
 akka.actor.allow-java-serialization = off
+
+akka.discovery.method = akka-dns
```

Comment:
Doesn't it need other configuration for Cluster Bootstrap? Should production configuration be split into another file like `prod-application.conf`?